api/lib/infrastructure/utils/pdf/certification-attestation-pdf.js

```diff
@@ -37,21 +41,42 @@ async function getCertificationAttestationsPdfBuffer({
   dirname = __dirname,
   fontkit = pdfLibFontkit,
   creationDate = new Date(),
+  i18n,
 } = {}) {
-  const viewModels = certificates.map((certificate) => AttestationViewModel.from(certificate, isFrenchDomainExtension));
+  const translate = i18n.__;
+  const lang = i18n.getLocale();
+
+  const viewModels = certificates.map((certificate) =>
+    AttestationViewModel.from({ certificate, isFrenchDomainExtension, translate, lang })
+  );
   const generatedPdfDoc = await _initializeNewPDFDocument(fontkit);
   generatedPdfDoc.setCreationDate(creationDate);
   generatedPdfDoc.setModificationDate(creationDate);
   const embeddedFonts = await _embedFonts(generatedPdfDoc, dirname);
   const embeddedImages = await _embedImages(generatedPdfDoc, viewModels);
 
-  const templatePdfPages = await _embedTemplatePagesIntoDocument(viewModels, dirname, generatedPdfDoc);
+  const templatePdfPages = await _embedTemplatePagesIntoDocument({
+    viewModels,
+    dirname,
+    pdfDocument: generatedPdfDoc,
+    lang,
+  });
 
-  await _render({ templatePdfPages, pdfDocument: generatedPdfDoc, viewModels, rgb, embeddedFonts, embeddedImages });
+  await _render({
+    templatePdfPages,
+    pdfDocument: generatedPdfDoc,
+    viewModels,
+    rgb,
+    embeddedFonts,
+    embeddedImages,
+    lang,
+  });
 
   const buffer = await _finalizeDocument(generatedPdfDoc);
 
-  const fileName = `attestation-pix-${moment(certificates[0].deliveredAt).format('YYYYMMDD')}.pdf`;
+  const fileName = `${translate('certification-confirmation.file-name')}-${dayjs(certificates[0].deliveredAt).format(
+    'YYYYMMDD'
+  )}.pdf`;
```

Comment:
Je propose de porter la concatenation par le translate pour éviter de faire de la concaténation pour toute chaine comprenant de l'internationalisation car [c'est une pratique non recommandée](https://www.w3.org/TR/international-specs/#strcat) 

```suggestion
 // Note: il faut du coup aAJouter le .pdf et un param dans les fichiers de trad
  const fileName = translate('certification-confirmation.file-name', {
    date: dayjs(certificates[0].deliveredAt).format(
    'YYYYMMDD'
  ),
  });
```